clients/fides-js/src/lib/consent-types.ts

```diff
@@ -120,9 +120,6 @@ export interface FidesInitOptions {
 
   // Shows fides.js overlay UI on load deleting the fides_consent cookie as if no preferences have been saved
   fidesClearCookie: boolean;
-
-  // Whether the developer forced the inclusion of the GPP extension via query param on the script tag
-  forceGpp: boolean;
```

Comment:
with this bug fix, GPP doesn't really need to care if forceGpp is on or not, it will now be correct no matter what.

Reply:
love that we have less overall code!